src/app/public/modules/radio/fixtures/radio-group-reactive.component.fixture.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Component,
+  ViewChild
+} from '@angular/core';
+
+import {
+  FormGroup,
+  FormBuilder,
+  FormControl
+} from '@angular/forms';
+
+import {
+  SkyRadioGroupComponent
+} from '../radio-group.component';
+
+@Component({
+  templateUrl: './radio-group-reactive.component.fixture.html'
+})
+export class SkyRadioGroupReactiveFixtureComponent {
```

Comment:
☝️ What's going on with these fixtures? Don't panic - `SkyRadioGroupReactiveFixtureComponent` appears to be a new file in this diff, but its actually the old fixture that was there before. I just renamed it to make way for a template-driven fixture. All our coverage was only using reactive forms. I created a new template-driven fixture to start covering TD forms. See `SkyRadioGroupFixtureComponent` below.